metadata.yaml

```diff
@@ -17,14 +17,11 @@ resources:
     filename: promethues-bind.exporter.snap
 
 requires:
-  juju-info:
-    interface: juju-info
-    scope: container
-
-provides:
   designate-bind:
     interface: bind-exporter
     scope: container
+
+provides:
   prometheus-bind-exporter:
```

Comment:
Since the charm is named `prometheus-bind-exporter`, I would recommend renaming this endpoint to `bind-exporter`. This is more of a nitpick, so feel free to ignore it.